extensions/gdx-setup/src/com/badlogic/gdx/setup/GdxSetupUI.java

```diff
@@ -432,6 +443,8 @@ public void actionPerformed (ActionEvent e) {
 		JLabel extensionsLabel = new JLabel("Extensions");
 		List<JPanel> extensionsPanels = new ArrayList<JPanel>();
 
+        SetupButton showMoreExtensionsButton = new SetupButton("Show More");
```

Comment:
It wouldn't be that large, and on a new line with reasonable padding away from the generate buttons it won't be a problem.  I don't want to hide it away, I want clear distinction :)